src/modules/UI/Wallets/reducer.js

```diff
@@ -248,11 +248,12 @@ function schema (wallet: AbcCurrencyWallet): GuiWallet {
   const currencyCode: string = wallet.currencyInfo.currencyCode
   const fiatCurrencyCode: string = wallet.fiatCurrencyCode.replace('iso:', '')
   const isoFiatCurrencyCode: string = wallet.fiatCurrencyCode
-  const symbolImage: string = wallet.currencyInfo.symbolImage
-  const symbolImageDarkMono: string = wallet.currencyInfo.symbolImageDarkMono
+  const symbolImage = wallet.currencyInfo.symbolImage
+  const symbolImageDarkMono = wallet.currencyInfo.symbolImageDarkMono
   const metaTokens: Array<AbcMetaToken> = wallet.currencyInfo.metaTokens
   const denominations: Array<AbcDenomination> = wallet.currencyInfo.denominations
-  const enabledTokens: Array<string> = wallet.enabledTokens || []
+  // TODO: Fetch the token list asynchonously before dispatching `schema`:
+  const enabledTokens: Array<string> = []
```

Comment:
How does this not break the app? enabledTokens will always be blank upon wallet initialization

Reply:
It was always broken. There was never any `wallet.enabledTokens` property, so this was always undefined, so the value was always an empty array. My change doesn't really affect anything; it just brings the existing problem to the forefront.